rules/testutil/testutil.go

```diff
@@ -39,10 +41,13 @@ func descriptorProtoFromSource(source io.Reader) (*descriptorpb.FileDescriptorPr
 	}
 	defer mustCloseAndRemoveFile(descSetF)
 
+	_, thisFilePath, _, _ := runtime.Caller(0)
```

Comment:
Will it work in Google3? Should we use the hack way as `protocPath`?

Reply:
Also make `testutil` an internal package -- it is no longer suitable for other libraries outside this repo to use it.

Reply:
This is a good point, and actually requires some restructuring that I've wanted to do for a while. I'm going to move this out of the `rules` package and into `api-linter/internal/testutil`.

I also think that the descriptor helpers should be in `api-linter/` and that `rules` should be strictly for `Rule` implementations and their tests, but we can decide that separately.

Reply:
I prefer to creating a new testing util package in and only for google3 -- as we already saw that the internal infrastructure is quite different. Otherwise, we will always have concerns about compatibility.

Reply:
Resolving because this code is now removed